leader-election-impl/src/test/java/com/palantir/leader/proxy/AwaitingLeadershipProxyTest.java

```diff
@@ -264,6 +267,53 @@ public void shouldClearDelegateUponLosingLeadership() throws Exception {
         verify(mock).close();
     }
 
+    @Test
+    public void whenMaybeNotCurrentLeaderAndLeadershipNotLostShouldNotClearDelegate() throws Exception {
+        MyCloseable mockA = mock(MyCloseable.class);
+        LeadershipCoordinator leadershipCoordinator = LeadershipCoordinator.create(leaderElectionService);
+        MyCloseable proxyA =
+                AwaitingLeadershipProxy.newProxyInstance(MyCloseable.class, () -> mockA, leadershipCoordinator);
+
+        // Wait to gain leadership
+        Uninterruptibles.sleepUninterruptibly(Duration.ofMillis(100L));
```

Comment:
Seems like the other tests have a `waitForLeadershipToBeGained()` call, is there a reason we can't/don't want to use that?

Reply:
good suggestion, this was simply from copying/pasting other tests 😬